phpBB/phpbb/notification/method/jabber.php

```diff
@@ -65,7 +65,7 @@ public function get_type()
 	*/
 	public function is_available(type_interface $notification_type = null)
 	{
-		return parent::is_available($notification_type) && $this->global_available() && $this->user->data['user_jabber'];
+		return parent::is_available($notification_type) && $this->global_available() && !empty($this->user->data['user_jabber']);
```

Comment:
I've changed this to use `!empty()` instead of `isset()` since this is IMHO the original intention of this line.